pkg/backup/abs_backend_test.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2017 The etcd-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backup
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/coreos/etcd-operator/pkg/backup/abs"
+)
+
+const (
+	absIntegrationTestEnvVar = "RUN_ABS_INTEGRATION_TEST"
```

Comment:
`RUN_ABS_INTEGRATION_TEST` -> `RUN_INTEGRATION_TEST`
I don't think we should have each ENV flag for each type of integration test.